### PR TITLE
port new missing block guardian script dev

### PR DESCRIPTION
**Pull request by dkijania**
Porting new missing guardian script #15384

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Review by emberian — APPROVED** [review]

File: scripts/deb-builder-helpers.sh [hunk 1]
```diff
@@ -331,21 +331,18 @@ build_archive_deb () {
...
-  cp ./default/src/app/berkeley_migration/berkeley_migration.exe "${BUILDDIR}/usr/local/bin/mina-berkeley-migration"
-  
-  mkdir -p "${BUILDDIR}/etc/mina/archive"
-  cp ../scripts/archive/download-missing-blocks.sh "${BUILDDIR}/etc/mina/archive"
-  
+
   cp ./default/src/app/missing_blocks_auditor/missing_blocks_auditor.exe "${BUILDDIR}/usr/local/bin/mina-missing-blocks-auditor"
   cp ./default/src/app/replayer/replayer.exe "${BUILDDIR}/usr/local/bin/mina-replayer"
   cp ./default/src/app/swap_bad_balances/swap_bad_balances.exe "${BUILDDIR}/usr/local/bin/mina-swap-bad-balances"
+  cp ../scripts/archive/missing-blocks-guardian.sh "${BUILD_DIR}/usr/local/bin/mina-missing-blocks-guardian"
```
Comment (emberian) [inline scripts/deb-builder-helpers.sh:338]:
```suggestion
  cp ../scripts/archive/missing-blocks-guardian.sh "${BUILDDIR}/usr/local/bin/mina-missing-blocks-guardian"
```

**Comment by emberian** [comment]
!ci-build-me

File: src/app/rosetta/docker-start.sh [hunk 2]
```diff
@@ -47,9 +47,12 @@ DEFAULT_FLAGS="--config-dir ${MINA_CONFIG_DIR} --peer-list-url ${PEER_LIST_URL}
...
 # Postgres database connection string and related variables
 POSTGRES_USERNAME=${POSTGRES_USERNAME:=pguser}
-POSTGRES_DBNAME=${POSTGRES_DBNAME:=archive}
+POSTGRES_DBNAME=${POSTGRES_DBNAME:=archive_balances_migrated}
+POSTGRES_PORT=${POSTGRES_PORT:=5432}
+POSTGRES_HOST=${POSTGRES_HOST:=127.0.0.1}
+POSTGRES_PASSWORD=${POSTGRES_PASSWORD:=pguser}
 POSTGRES_DATA_DIR=${POSTGRES_DATA_DIR:=/data/postgresql}
-PG_CONN=postgres://${POSTGRES_USERNAME}:${POSTGRES_USERNAME}@127.0.0.1:5432/${POSTGRES_DBNAME}
+PG_CONN=postgres://${POSTGRES_USERNAME}:${POSTGRES_PASSWORD}@${POSTGRES_DATA_DIR}:${POSTGRES_PORT}/${POSTGRES_DBNAME}
```
Comment (kaozenn) [inline src/app/rosetta/docker-start.sh:55]: `127.0.0.1` was replaced by `${POSTGRES_DATA_DIR}`

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Pull request merged**